src/components/forms/input.jsx

```diff
@@ -1,22 +1,27 @@
-var React = require('react');
 var classNames = require('classnames');
+var FRCInput = require('formsy-react-components').Input;
+var React = require('react');
+var defaultValidationHOC = require('./validations.jsx').defaultValidationHOC;
+var inputHOC = require('./input-hoc.jsx');
 
 require('./input.scss');
+require('./row.scss');
 
 var Input = React.createClass({
     type: 'Input',
-    propTypes: {
-        
+    getDefaultProps: function () {
```

Comment:
Is this functionally equivalent to simply removing the `getDefaultProps` attribute?